readme.md

```diff
@@ -354,14 +354,14 @@ ReactDom.render(
         const match = /language-(\w+)/.exec(className || '')
         return !inline && match ? (
           <SyntaxHighlighter
+            {...props}
```

Comment:
It might be better to not even pass spread props at all.

Anyway, I'm not necessarily against it.

Reply:
I think there was something about `SyntaxHighlighter` passing all other props through on the `div` or so:

> spread props pass arbitrary props to pre tag wrapping code.
> — https://github.com/react-syntax-highlighter/react-syntax-highlighter/tree/efc3f7b7537d1729193b7a472067bcbe6cbecaf1#props

Could be removed too, okay with me. But that can be a different PR!